llvm/docs/DeveloperPolicy.rst

```diff
@@ -527,6 +527,21 @@ after they are committed, depending on the nature of the change).  You are
 encouraged to review other peoples' patches as well, but you aren't required
 to do so.
 
+Triage Role
```

Comment:
It seems like this should be probably be reviewed in a separate patch/maybe accompanied by a post on Discourse given it's more of a project policy thing? At least the second sentence of the first paragraph.

The second paragraph can probably just be landed, but probably in a separate commit to keep things cleanish.

Reply:
OK, I'll do a separate PR with the doc changes.